onnxruntime/__init__.py

```diff
@@ -76,3 +76,80 @@
     __version__ = version
 
 onnxruntime_validation.check_distro_info()
+
+# Load nvidia libraries from site-packages/nvidia if the package is onnxruntime-gpu
```

Comment:
Why do we need to do so?

Reply:
In case we decided to change the name of the cuda packge to onnxruntime-cuda